src/lambda/explorer/regionNode.ts

```diff
@@ -24,23 +27,29 @@ export class RegionNode extends AWSRegionTreeNode {
     }
 
     public async getChildren(): Promise<AWSTreeNodeBase[]> {
-        const lambdaFunctions: AWSTreeNodeBase[] = await getLambdaFunctionsForRegion(this.regionCode)
+        const lambdaFunctions: FunctionNode[] = await getLambdaFunctionsForRegion(this.regionCode)
 
         if (lambdaFunctions.length === 0) {
-            lambdaFunctions.push(new NoFunctionsNode(
-                localize('AWS.explorerNode.lambda.noFunctions', '[no functions in this region]'),
-                'awsLambdaNoFns'
-            ))
+            return [new NoFunctionsNode(
```

Comment:
It's possible that a user has CloudFormation Stacks to show, and not Lambda Functions. It's also possible that we show additional CloudFormation stack resources in addition to Lambda Functions in the future. There shouldn't be a lambdaFunction length check here, so that we always display the Lambda/CloudFormation nodes (below), even if they report "no functions/stacks" as their children.

Reply:
It makes total sense to me. Can the generic node report "no items"? We can pass a placeholder as an argument if necessary.

Reply:
Yes, I like that the generic node is able to report "no items"